### PR TITLE
[common-lisp/en] Fix code in examples

### PR DESCRIPTION
**Pull request by davydovanton**

### CODE REVIEW

File: common-lisp.html.markdown [hunk 1]
```diff
@@ -430,7 +430,7 @@ nil                  ; for false - and the empty list
 (defun walker (n)
   (if (zerop n)
       :walked
-      (walker (1- n))))
```
Comment (vendethiel) [inline common-lisp.html.markdown]: That's correct

**Comment by adambard** [comment]
Ty!

**Comment by vendethiel** [comment]
There was no need to pull in the other change. Oh well, not a big deal.

**Comment by adambard** [comment]
Sorry, I'm just trying to get through PR mountain a little bit. I figure an improvement and a doesn't matter is with merging.

**Comment by vendethiel** [comment]
Definitely.

**Pull request merged**